glean-core/ios/sdk_generator.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Fail the script on first error
```

Comment:
Whoops, sorry if I was unclear here. I really meant to add a comment about what this file is used for.